app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -49,7 +49,9 @@ module ContentFacetHostExtensions
         )
 
         def content_facet_ignore_update?(attributes)
-          self.content_facet.blank? && attributes.blank?
+          self.content_facet.blank? &&
+            attributes.values.uniq.length == 1 &&
+            attributes.values.uniq.first.blank?
```

Comment:
@dLobatog i think what i suggested is more clear about what is going on that this is.